config/install/user.role.az_self_service_editor.yml

```
@@ -0,0 +1,25 @@
+langcode: en
+status: true
+dependencies: {  }
+id: az_self_service_editor
+label: 'Self-service editor'
```

```suggestion
label: 'Self-service profile editor'
```

This is the best suggestion I can think for for a more specific role name.